toolkits/global/packages/global-expander/__tests__/expander.spec.js

```diff
@@ -328,12 +328,12 @@ describe('Expander', () => {
 			expect(element.BUTTON.textContent).toEqual(triggerOpenLabelOption);
 		});
 
-		test('Should use CLOSE_ON_CLICKOFF option if it is passed to constructor', () => {
+		test('Should close target if target is clocked off if CLOSE_IF_NOT_FOCUSED option used', () => {
```

Comment:
Why change the format of this test title? Compared to all the others which use the `Should use XXX_XXX option if it is passed to constructor`

Reply:
I just found it more descriptive as to what the test is trying to achieve and there's no having to imply or insinuate as much. Happy to keep it like the others though

Reply:
The behaviour is actually tested earlier (`test('Should close when click off the target', () => {`. I wanted initially to just make an assertion that the correct option was passed to the constructor but it didn't assert that the option changed something. So I ended up with this, check the main behaviour, then check the option.

You are right that it doesn't describe what it does, but in that case we should change all the option based test titles.

Reply:
Ah ok, I hadn't seen the other test. Maybe this test isn't needed then as the other test actually already tests that the option is passed to the constructor?

Reply:
Yeah, problem is that in the case of focus out, we initially test that it closes when you focus out because the option is true by default. But it doesn't test that it doesn't close if the option is false.
Perhaps the tests should be reorganised a bit. I tried to group them logically but it's not quite right.

Reply:
So we could do:

```
test('Should close when focus out of the target', () => { ... });
test('Should NOT close when focus out of the target if CLOSE_ON_FOCUS_OUT is false', () => { ... });
```

And reorganised the others that have options similarly.

Reply:
Sounds like a separate PR to me?

Reply:
I think they're fine tbh, unless you have a particular structure you want in mind?

Reply:
Not urgently 👍